big_tests/tests/offline_SUITE.erl

```diff
@@ -153,6 +189,16 @@ expired_messages_are_not_delivered(Config) ->
 is_chat(Content) ->
     fun(Stanza) -> escalus_pred:is_chat_message(Content, Stanza) end.
 
+is_groupchat(Content) ->
+    fun(Stanza) -> escalus_pred:is_groupchat_message(Content, Stanza) end.
```

Comment:
Do we want to check if the groupchat message comes from offline? If it comes from offline, it has a `delayed` tag as described in the XEP.

Reply:
yes, we can also check this tag for chat and affiliation messages.